lib/cocoapods/installer/analyzer/sandbox_analyzer.rb

```diff
@@ -43,11 +43,7 @@ class SandboxAnalyzer
         alias_method :update_mode?, :update_mode
 
         # @return [Lockfile] The lockfile of the installation as a fall-back if
-        #         there is no sandbox manifest. This is indented as a temporary
-        #         solution to prevent the full re-installation from users which
-        #         are upgrading from CP < 0.17.
-        #
-        # @todo   Remove for CP 0.18.
```

Comment:
maybe we can remove this now ;)

Reply:
Haha good point. I can do that in another PR?

Reply:
🎊